addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -91,11 +92,17 @@ def _l10n_it_edi_prepare_fatturapa_line_details(self, reverse_charge_refund=Fals
                 price_unit = line.price_unit
 
             description = line.name
-            if not is_downpayment:
-                if line.price_subtotal < 0:
-                    moves = line._get_downpayment_lines().move_id
-                    if moves:
-                        description += ', '.join([move.name for move in moves])
+
+            # Down payment lines:
+            # If there was a down payed amount that's been deducted from this move,
+            # we need to put a reference to the down payment invoice in the DatiFattureCollegate tag
+            downpayment_moves = self.env['account.move']
+            if not is_downpayment and line.price_subtotal < 0:
+                downpayment_moves = line._get_downpayment_lines().mapped("move_id")
+                if downpayment_moves:
+                    downpayment_moves_description = ', '.join([move.name for move in downpayment_moves])
+                    sep = ', ' if description else ''
```

Comment:
is not line.name a required field anyways?

Reply:
No, can be empty

![image](https://user-images.githubusercontent.com/1665365/229874931-f9eccc76-3154-4ead-9e96-4b4539a4698f.png)
![image](https://user-images.githubusercontent.com/1665365/229874952-c67d4385-4b27-463f-8b4b-4ba9471d31cb.png)